m4/with_pmix.m4

```diff
@@ -0,0 +1,88 @@
+
+#
+# Copyright (C) 1994-2019 Altair Engineering, Inc.
+# For more information, contact Altair at www.altair.com.
+#
+# This file is part of the PBS Professional ("PBS Pro") software.
+#
+# Open Source License Information:
+#
+# PBS Pro is free software. You can redistribute it and/or modify it under the
+# terms of the GNU Affero General Public License as published by the Free
+# Software Foundation, either version 3 of the License, or (at your option) any
+# later version.
+#
+# PBS Pro is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+# FOR A PARTICULAR PURPOSE.
+# See the GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Commercial License Information:
+#
+# For a copy of the commercial license terms and conditions,
+# go to: (http://www.pbspro.com/UserArea/agreement.html)
+# or contact the Altair Legal Department.
+#
+# Altair’s dual-license business model allows companies, individuals, and
+# organizations to create proprietary derivative works of PBS Pro and
+# distribute them - whether embedded or bundled with other software -
+# under a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+#
+
+AC_DEFUN([PBS_AC_WITH_PMIX],
+[
+  AC_ARG_WITH([pmix],
+    AS_HELP_STRING([--with-pmix=DIR],
```

Comment:
Tried to download the source and configured it with: ./configure --with-pmix=/opt/pmix after I've installed PMIX, and make failed with:

make[2]: Leaving directory `/home/bayucan/git/pbspro/src/modules'
Making all in resmom
make[2]: Entering directory `/home/bayucan/git/pbspro/src/resmom'
gcc -DHAVE_CONFIG_H -I. -I../../src/include  -DPBS_MOM -I../../src/include -I../../src/resmom/linux -I/usr/include/python3.6m -I/usr/include/python3.6m     -g -O2 -MT pbs_mom-mom_start.o -MD -MP -MF .deps/pbs_mom-mom_start.Tpo -c -o pbs_mom-mom_start.o `test -f 'linux/mom_start.c' || echo './'`linux/mom_start.c
In file included from linux/mom_start.c:69:0:
./mom_pmix.h:52:25: fatal error: pmix_server.h: No such file or directory
 #include <pmix_server.h>